lib/seahorn/BvOpsem2WideMemMgr.cc

```diff
@@ -542,13 +538,29 @@ class WideMemManager : public OpSemMemManagerBase {
     LOG("opsem", WARN << "setFatData() not implemented!\n");
     return nullPtr();
   }
+
+  Expr isDereferenceable(PtrTy p, Expr byteSz) {
+    // size should be >= byteSz
+    if (m_ctx.alu().isNum(byteSz) && m_ctx.alu().isNum(p.getSize())) {
+      signed numBytes = m_ctx.alu().toNum(byteSz).get_si();
```

Comment:
these should be unsigned. Perhaps we need `get_ui()` in ALU for that.
Probably does not matter in practice since we expected these sizes to be fairly small